app/scripts/controllers/next-release.ts

```diff
@@ -12,10 +12,13 @@ module labsFrontendApp
     export class NextReleaseCtrl
     {
         // @ngInject
-        constructor (private $scope: INextReleaseScope, private releases: GetAllPublicReleases)
+        constructor (private $scope: INextReleaseScope, private cachedReleases: GetAllPublicReleases)
```

Comment:
I needed to have a different variable name, injection works based on variable names here :S